codelists/codelists.txt

```diff
@@ -0,0 +1,2 @@
+opensafely/ethnicity/2020-04-27
```

Comment:
Currently we are loading the ethnicity codelist but not using it. For example studies we probably want to avoid circular dependencies of actions using other actions? So might we want to avoid using the cohort joiner in the decile chart action example study? 
Since it is a small example study, it probably would not be too expensive to extract ethnicity for each month.
Or we could assume a reusable action hierarchy in which decile charts could use the cohort joiner, but the cohort joiner would not use decile charts.